test/JDBC/input/datatypes/TestDatetime-vu-verify.txt

```diff
@@ -1 +1,35 @@
 SELECT * FROM TestDatetime_vu_prepare_dt;
+
+select convert(datetime, '2022-11-10 09:15:30.000');
+select convert(datetime, '2022-11-10 09:15:30.001');
+select convert(datetime, '2022-11-10 09:15:30.002');
+select convert(datetime, '2022-11-10 09:15:30.003');
+select convert(datetime, '2022-11-10 09:15:30.004');
+select convert(datetime, '2022-11-10 09:15:30.005');
+select convert(datetime, '2022-11-10 09:15:30.006');
+select convert(datetime, '2022-11-10 09:15:30.007');
+select convert(datetime, '2022-11-10 09:15:30.008');
+select convert(datetime, '2022-11-10 09:15:30.009');
+select convert(datetime, '2022-11-10 09:15:30.999');
+
+select convert(datetime, '2022-11-10 21:26:45.120');
+select convert(datetime, '2022-11-10 21:26:45.341');
+select convert(datetime, '2022-11-10 21:26:45.562');
+select convert(datetime, '2022-11-10 21:26:45.783');
+select convert(datetime, '2022-11-10 21:26:45.904');
+select convert(datetime, '2022-11-10 21:26:45.215');
+select convert(datetime, '2022-11-10 21:26:45.436');
+select convert(datetime, '2022-11-10 21:26:45.657');
+select convert(datetime, '2022-11-10 21:26:45.878');
+select convert(datetime, '2022-11-10 21:26:45.909');
+
+select convert(datetime, '2000-12-13 12:58:23.123');
+select convert(datetime, '2000-12-13 12:58:23.223');
+select convert(datetime, '2000-12-13 12:58:23.323');
+select convert(datetime, '2000-12-13 12:58:23.423');
+select convert(datetime, '2000-12-13 12:58:23.523');
+select convert(datetime, '2000-12-13 12:58:23.623');
+select convert(datetime, '2000-12-13 12:58:23.723');
+select convert(datetime, '2000-12-13 12:58:23.823');
+select convert(datetime, '2000-12-13 12:58:23.923');
+select convert(datetime, '2000-12-13 12:58:23.023');
```

Comment:
We should add more details to test template about actual use cases not just done/NA. Also, why are are saying boundary conditions or negative values are not applicable. We can use lowest data, 0:0:0 and invalid date input etc unless covered already

Reply:
Negative dates not applicable.
Boundary condition / Datetime valid range: ('1753-01-01' To '9999-12-31')
Have added testcases with values inside and outside the valid range.